src/extractcode/extract.py

```diff
@@ -176,7 +176,7 @@ def extract_files(location, kinds=extractcode.default_kinds, recurse=False):
             if recurse:
                 if TRACE:
                     logger.debug('extract:walk: recursing on target: %(target)r' % locals())
-                for xevent in extract(target, kinds, recurse):
+                for xevent in extract(target, ignored_extensions, kinds, recurse):
```

Comment:
Here and elsewhere I think using named kwargs is usually better and a more robust approach: so it is fine to apply that change as in:
`for xevent in extract(target=target, kinds=kinds, recurse=recurse, ignored_extensions=ignored_extensions):`
both here and elsewhere you made changes